mlos_bench/mlos_bench/storage/sql/schema.py

```diff
@@ -144,6 +144,19 @@ def __init__(self, engine: Engine):
                                  [self.trial.c.exp_id, self.trial.c.trial_id]),
         )
 
+        self.trial_status = Table(
+            "trial_status",
+            self._meta,
+            Column("exp_id", String(255), nullable=False),
+            Column("trial_id", Integer, nullable=False),
+            Column("ts", DateTime, nullable=False, default="now"),
+            Column("status", String(16), nullable=False),
```

Comment:
Maybe also `enviroment_name` or `environment_path` (e.g., include the parent's name as well to account for disambiguation since we don't enforce uniqueness)?

Reply:
The plan is to use that table to store the status updates and also use it as a scheduling queue. That is, we'll have records with a status `SCHEDULED` + some future timestamp, and also add a record to this table each time when we poll the environment for status. I would rather keep this table's schema very minimal for that reason

Reply:
I've removed this change from the PR